src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -488,6 +490,18 @@ protected void setup()
         completeSetup();
     }
 
+    public void installCryptoProvider()
```

Comment:
As mentioned above, this is not necessary. You can install the provider as part of `DatabaseDescriptor.applyAll`